src/main/java/com/softserve/academy/spaced/repetition/service/impl/CategoryServiceImpl.java

```diff
@@ -20,41 +21,44 @@
 public class CategoryServiceImpl implements CategoryService {
 
     private final int QUANTITY_CATEGORIES_IN_PAGE = 8;
+    private final String ALL_CATEGORIES_CACHE_NAME = "allCategories";
+    private final String TOP_CATEGORIES_CACHE_NAME = "topCategories";
 
     @Autowired
     private CategoryRepository categoryRepository;
     @Autowired
     private ImageService imageService;
 
+
     @Override
-    @Transactional
+    @Cacheable(ALL_CATEGORIES_CACHE_NAME)
```

Comment:
did you write cashing only for category?
what about decks?

Reply:
decks cache will be much bigger